### DISCUSSION TITLE
Why after run `pnpm add express`, the `mime-types` in the first level directoy of node_modules?

### DISCUSSION THREAD

**Discussion by jawn-ha** [pnpm/pnpm#4528] [Q&A] [answered]
the express version is 4.17.3
the pnpm version is 6.24.2
After run `pnpm add express`
I found express is not the only one of the first level directoy of node_modules，but `mime-types`
I hope someone who can help me explain this
<img width="303" alt="image" src="https://user-images.githubusercontent.com/16000122/161501133-ae80784b-b879-4810-8632-47d3dec08d57.png">

**Comment by zkochan** [comment, marked as answer]
Because by default pnpm hoists any packages that have "types" in their names.

https://pnpm.io/6.x/npmrc#public-hoist-pattern

This will change in pnpm v7. In v7, types will not be hoisted by default.
